src/components/node/person_profile/index.tsx

```diff
@@ -0,0 +1,118 @@
+import Image from '@/components/image'
+import { DEV_PATH } from '@/lib/constants'
```

Comment:
I think this is fine for completing the component but I'd like to at least mark one of the uses of `DEV_PATH` here for removal, maybe with a simple `@todo`. We'll want to handle image paths differently eventually.